keps/sig-scheduling/2458-node-resource-score-strategy/README.md

```diff
@@ -313,45 +315,49 @@ type RequestedToCapacityRatio struct {
 // Note that the two types defined below already exist in the scheduler's component config API.
 // ResourceSpec represents a single resource.
 type ResourceSpec struct {
-	// Name of the resource.
-	Name string
-	// Weight of the resource.
-	Weight int64
+    // Name of the resource.
+    Name string
+    // Weight of the resource.
+    Weight int64
 }
 
 // UtilizationShapePoint represents a single point of a priority function shape.
 type UtilizationShapePoint struct {
-	// Utilization (x axis). Valid values are 0 to 100. Fully utilized node maps to 100.
-	Utilization int32
-	// Score assigned to a given utilization (y axis). Valid values are 0 to 10.
-	Score int32
+    // Utilization (x axis). Valid values are 0 to 100. Fully utilized node maps to 100.
+    Utilization int32
+    // Score assigned to a given utilization (y axis). Valid values are 0 to 10.
+    Score int32
 }
 
 ```
 
-Add `PreferredFitStrategy` to `NodeResourcesFitArgs`:
+Add `ScoringStrategy` to the existing `NodeResourcesFitArgs`:
 
 ```go
 
 // NodeResourcesFitArgs holds arguments used to configure the NodeResourcesFit plugin.
 type NodeResourcesFitArgs struct {
-	metav1.TypeMeta
+    metav1.TypeMeta
 
-	// IgnoredResources is the list of resources that NodeResources fit filter
-	// should ignore.
-	IgnoredResources []string
-	// IgnoredResourceGroups defines the list of resource groups that NodeResources fit 
+    // IgnoredResources is the list of resources that NodeResources fit filter
+    // should ignore.
+    IgnoredResources []string
+    // IgnoredResourceGroups defines the list of resource groups that NodeResources fit 
     // filter should ignore.
-	// e.g. if group is ["example.com"], it will ignore all resource names that begin
-	// with "example.com", such as "example.com/aaa" and "example.com/bbb".
-	// A resource group name can't contain '/'.
-	IgnoredResourceGroups []string
+    // e.g. if group is ["example.com"], it will ignore all resource names that begin
+    // with "example.com", such as "example.com/aaa" and "example.com/bbb".
+    // A resource group name can't contain '/'.
+    IgnoredResourceGroups []string
     
-    // PreferredFitStrategy selects the node resource scoring strategy.
-    PreferredFitStrategy *PreferredFitStrategy
+    // ScoringStrategy selects the node resource scoring strategy.
+    ScoringStrategy *ScoringStrategy
 }
 ```
 
+
+As of writing this KEP, scheduler component config is v1beta1. The plugins we plan to
+deprecate will continue to be configurable in v1beta1, and will be deprecated in v1beta2. 
```

Comment:
"and will not be available in v1beta2"?

"deprecated" would mean that people can still use it, even though we plan to remove it.

Reply:
done.